src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Consider using a more descriptive name that reflects the purpose of the variable, such as `styleObject`.